UniversalAlgebra/GaloisConnections/Properties.lagda

```diff
@@ -58,20 +58,26 @@ module _ {α ρ : Level}{𝒜 : Type α} where
          ; antisym = _,_
          }
 
+\end{code}
 
-module _ {ℓ ρ : Level}{𝒜 ℬ : Type ℓ} where
+A Binary relation from one poset to another induces a Galois connection, but only in a very special
+situation, namely when all the involved sets are of the same size.  This is akin to the situation
+with Adjunctions in Category Theory (unsurprisingly). In other words, there is likely a
+unit/counit definition that is more level polymorphic.
```

Comment:
Nice explanation!

Here again is one of the pleasant side effects of formalizing... we are forced to notice things that are glossed over, or even overlooked, in informal treatments.